docs/core/index.yml

```diff
@@ -0,0 +1,122 @@
+### YamlMime:Landing
+
+title: .NET Core documentation
+summary: Learn about .NET Core, an open-source, general-purpose development platform for building cross-platform apps.
+
+metadata:
+  title: .NET Core documentation
+  description: .NET Core is an open-source, general-purpose development platform for building cross-platform apps.
+  ms.topic: landing-page
+  ms.date: 03/24/2020
+
+# linkListType: architecture | concept | deploy | download | get-started | how-to-guide | learn | overview | quickstart | reference | sample | tutorial | video | whats-new
+
+landingContent:
+
+  # Card
+  - title: Learn about .NET Core
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: About .NET Core
+            url: about.md
+          - text: NET Core SDK overview
+            url: sdk.md
+      - linkListType: whats-new
+        links:
+          - text: What's new in .NET Core 3.1
+            url: whats-new/dotnet-core-3-1.md
+          - text: What's new in .NET Core 3.0
+            url: whats-new/dotnet-core-3-0.md
+
+  # Card
+  - title: Get started with .NET Core
+    linkLists:
+      - linkListType: overview
+        links:
+          - text: Migrate from .NET Framework to .NET Core
```

Comment:
This is an overview but it's not an overview of getting started with .NET Core. It might be worth putting this in its own "Migrate from..." card